plat/layerscape/board/ls1043/drivers/uart/16550_console.S

```diff
@@ -0,0 +1,294 @@
+/*
+ * Copyright (c) 2018, ARM Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <arch.h>
+#include <asm_macros.S>
+#include <console_macros.S>
+#include <assert_macros.S>
+
+/* UART16550 Registers */
```

Comment:
Thanks for your comments, but layerscape uart register definition is different from TI's uart, for example,
Layersacpe:
 #define UARTIER			0x1
But TI:
#define UARTIER			0x4
So we can't reuse TI's header file, thanks.

Reply:
OK, Are the definitions of console APIs here different from the TI one ?

Reply:
Hi, Soby, 

In fact there is no difference from APIs with TI one. but Layerscape's uart register offset is aligned with byte:
#define UARTDLL                 0x0
#define UARTIER                 0x1
#define UARTDLLM                0x1
#define UARTFCR                 0x2
#define UARTLCR                 0x3
#define UARTMCR                 0x4
#define UARTLSR                 0x5
But TI's uart register is aligned with word:
#define UARTDLL                 0x0
#define UARTIER                 0x4
#define UARTDLLM                0x4
#define UARTIIR                 0x8
#define UARTFCR                 0x8
#define UARTLCR                 0xc

In TI's uart driver, it use some word instruction, such as ldr, str for register accessing, but layerscape's uart driver has to use byte instruction, such as ldrb, strb.

So we can't reuse TI's uart driver directly, this is the reason why we would like to keep a private uart driver. 

Thanks.
Jiafei.

Reply:
OK, I see. In that case, I would prefer if the functions here use a prefix to avoid name clash with existing driver. Perhaps prefix all functions with `ls_` i.e console_16550_core_init be renamed to ls_console_16550_core_init and so on..